src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/FixAllContextHelper.cs

```diff
@@ -46,6 +46,7 @@ public static async Task<ImmutableDictionary<Document, ImmutableArray<Diagnostic
                 // Note: We avoid fixing diagnostics in generated code.
                 if (document != null && !await document.IsGeneratedCodeAsync(cancellationToken).ConfigureAwait(false))
                 {
+#if WORKSPACE
```

Comment:
Should we advertise in some why that this helper will not work for these scopes from the code style layer? DocComment and Debug.Assert perhaps?

Reply:
so, practically, it's a non issue.  Here's why:

1. we changed it so that we do NOT use the code-style analyzers/fixers ourselves within VS.  Jared updated things so we always use what we shipped in box.
2. So this only affects code style analyzers/fixers run by other tools that are pulling that in (like `dotnet format`).  Those tools are not doing fix-all in that scoped fashion ('in correct type', etc).  

I'm happy to comment and add asserts.  I have a followup pr i can put that in if that works for you.